README.md

```diff
@@ -122,7 +126,7 @@ EOF
     return false -- can't save
   end,
   write_all_buffers = false, -- write all buffers when the current one meets `condition`
-  debounce_delay = 135, -- saves the file at most every `debounce_delay` milliseconds
+  debounce_delay = 135, -- delay after which a pending save is executed
```

Comment:
To me this sounds a very short default value now with the `defer_save` and `cancel_save`. I'd go as far as set it two something like 2000ms but I'll do that first on my own config and I'll report back on how it works.

Reply:
Totally true! I actually have set it to 2000ms on my local config as well.
I'll toy around with it a bit, I don't see why we shouldn't go with 2000ms.

Reply:
I went with 1000ms for now :)